pkg/metadata/api_info.go

```diff
@@ -11,19 +11,16 @@
 // express or implied. See the License for the specific language governing
 // permissions and limitations under the License.
 
-package multiversion
-
-// TODO(a-hilaly) move this file outside of pkg/model/multiversion. Idealy we
-// Should be able to access APIStatus and APIInfo to prevent regenerating removed or
-// deprecated APIs.
+package metadata
 
 type APIStatus string
 
 const (
-	APIStatusUnknown    APIStatus = "unknown"
-	APIStatusAvailable            = "available"
-	APIStatusRemoved              = "removed"
-	APIStatusDeprecated           = "deprecated"
+	APIStatusUnknown       APIStatus = "unknown"
+	APIStatusInDevelopment           = "in_development"
```

Comment:
Can we call this just `development` and make it the default instead of `unknown`?

Reply:
I'm wondering if we really need a `development` status for API versions. I think that we should increase the API Version whenever the CRD schema changes.

Reply:
IMO `development` should be a status for the service controller and not a specific API version.

Reply:
That seems fair. The API version in development will always be the latest available option. "In development" vs. "released" is controlled by our controller release already.